### ISSUE TITLE
Implement Design

### ISSUE THREAD

**Issue by octalmage** [octalmage/Marknote#9]
This issue is to track the design progress.

**Comment by octalmage** [comment]
Do you still like the dog ear in the design above (with the #)? @mchamplin

I want to try it, do you still have the related graphics?

**Comment by mjchamplin** [comment]
Probably. I want to get back involved in this project, so if I have to redraw it / start from scratch I will. ;)

**Comment by octalmage** [comment]
Awesome! yeah I'm trying to get back into it also.

**Comment by mjchamplin** [comment]
Nice icon! Did you draw it / can I tweak a couple things? 

Also I'm moving in 10 days but after that I'd really love to dedicate some time to the design of this app. What would be a good way for us to catch up? Google Hangout or something?

**Comment by octalmage** [comment]
Yeah do whatever you want! I just got tired of looking at the nw.js icon so I made something in Sletch, but I'm not attached to it one bit.

And yeah Google Hangouts works! Just let me know!

**Comment by mjchamplin** [comment]
Awesome! 
 
I probably won't wear pants though.

**Comment by octalmage** [comment]
That's to be expected.

**Comment by mjchamplin** [comment]
Hey dude, I'm all moved and I'd love to work on Marknote again. Let me know when you'd be free for a little chatty chat.

**Comment by octalmage** [comment]
Awesome! I'm off the next couple of days. So whenever!

**Comment by MartinDesigner** [comment]
![imghome](https://cloud.githubusercontent.com/assets/13809960/9289329/0bef602c-436c-11e5-80fc-3ec07f82dd8d.jpg)
So, as a supplement, however as a main point try this amazing free app. Createer.com is an online software for easily designing online ads, responsive landing pages and print materials such as flyers, posters, covers, invitations and more. Createer lets you easily share your designs on social media, send them via email, download as HTML, image or PDF. Createer also includes 3,000 predesigned templates (browse here: https://beta2.createer.com/Templates?ghi) and over 250,000 images (illustrations, photos and icons) to work with in its user-friendly drag’n’drop enviroment. And finally this amazing stuff is here: https://beta2.createer.com/?ghi

**Comment by mjchamplin** [comment]
Github spam! That's new...

**Comment by octalmage** [comment]
Lol yeah, do they even have anti spam tools?

**Comment by mjchamplin** [comment]
I reported the account...

**Comment by mjchamplin** [comment]
Hey! Would love to revisit this and update / provide designs if needed.
